python/orca/test/bigdl/orca/learn/ray/pytorch/test_estimator_ray_backend.py

```diff
@@ -396,7 +396,10 @@ def test_pandas_dataframe(self):
         estimator.evaluate(data_shard, batch_size=2, feature_cols=["user", "item"],
                            label_cols=["label"])
         result = estimator.predict(data_shard, batch_size=2, feature_cols=["user", "item"])
-        result.collect()
+        predictions = result.collect()[0]
+        import pandas as pd
+        assert isinstance(predictions, pd.DataFrame), "predict should return a pandas dataframe"
+        print(predictions.head(n=1))
```

Comment:
Two choices:
+ use `isinstance`, need to import pandas
+ use method `.head()` to test predictions is a pdf not a dict

Reply:
isinstance should be better, since ut is mainly provided for the automatic test.